include/erlcloud_ecs.hrl

```diff
@@ -247,21 +259,51 @@
     task_role_arn :: undefined | binary() 
 }).
 
+-record(ecs_attachment_detail, {
+    name :: undefined | binary(),
+    value :: undefined | binary()
+}).
+
+-record(ecs_attachment, {
+    id :: undefined | binary(),
+    type :: undefined | binary(),
+    status :: undefined | binary(),
+    details :: undefined | [#ecs_attachment_detail{}]
+}).
+
+-record(ecs_tag, {
+    key :: undefined | binary(),
+    value :: undefined | binary()
+}).
+
 -record(ecs_task, {
+    attachments :: undefined | [#ecs_attachment{}],
+    availability_zone :: undefined | binary(),
     cluster_arn :: undefined | binary(),
+    connectivity :: undefined | binary(),
+    connectivity_at :: undefined | number(),
     container_instance_arn :: undefined | binary(),
     containers :: undefined | [#ecs_container{}],
+    cpu :: undefined | binary(),
     created_at :: undefined | number(),
-    desired_status:: undefined | binary(),
+    desired_status :: undefined | binary(),
+    group :: undefined | binary(),
+    health_status :: undefined | binary(),
     last_status :: undefined | binary(),
     launch_type :: undefined | binary(),
+    memory :: undefined | binary(),
     overrides :: undefined | #ecs_task_override{},
-    started_at :: undefined | pos_integer(),
+    platform_version :: undefined | binary(),
+    pull_started_at :: undefined | number(),
+    pull_stopped_at :: undefined | number(),
+    started_at :: undefined | number(),
     started_by :: undefined | binary(),
-    stopped_at :: undefined | pos_integer(),
+    stopped_at :: undefined | number(),
     stopped_reason :: undefined | binary(),
+    tags :: undefined | [#ecs_tag{}],
     task_arn :: undefined | binary(),
-    task_definition_arn :: undefined | binary()
+    task_definition_arn :: undefined | binary(),
+    version :: undefined | pos_integer()
```

Comment:
This one is a `Long` in the [docs](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Task.html#ECS-Type-Task-version). Is it right that it's an integer, always positive?

Reply:
I haven't seen negative version in real life. And doc says

> Every time a task experiences a change that triggers a CloudWatch event, the version counter is incremented

I think it is safe to assume it is non negative